.rat-excludes

```diff
@@ -109,6 +109,9 @@ TCPIPV6_os.*
 .ism*
 resource.h*
 resource.hm*
+odb.sln
+odb.vcxporj*
```

Comment:
Is this line a typo? (I notice that the next line is odb.vcxproj*). I don't see any files that match this pattern. Perhaps we can remove this line?